cmd/run.go

```diff
@@ -90,6 +90,10 @@ Non-bundle data file and directory paths can be prefixed with the desired
 destination in the data document with the following syntax:
 
 	<dotted-path>:<file-path>
+		
+To set a data file as the input document in the REPL prefix the file path:
+		
+	opa run repl.input:input.json
```

Comment:
Lets add a pointer to the help doc too, something like:
```
Use the "help" command in the interactive shell to see more options
```

Reply:
@patrick-east , thanks for you precious reviews , I fixed that at commit 3945a1d.

Reply:
@patrick-east , thanks for you precious reviews , I fixed that at commit 3945a1d.